CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+First, we would like to thank you for taking the time to contribute to this project. We really appreciate it. We hope to make this a pleasant and fun experiece for you!
+
+When contributing to this repository, please first discuss the change you wish to make via the **issue comments** you are interested in working on, before making any changes.
+
+Please note and visit our [Code of Conduct here!](https://github.com/Github-Community-Standards/github-community-standards/blob/main/CODE_OF_CONDUCT.md "Github-Community-Standards Code of Conduct"). Thank you in advance for being sure to follow it in all your interactions with the project.
+
+## Claiming/Assinging an Issue:
+
+1. First, **_always verify_** that the issue is not already assigned to anyone.
+1. If you are a [Collaborator](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository "Github Collaborator Docs"), you may assign the issue to yourself.
+1. If you are a [Contributor](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors "Github Contributor Docs"), you may request the issue be assigned to you.
+1. You may begin working on the issue once the issue has been assigned to you.
+1. Be sure to attach **_all related issues_** to the appropriate pull request using [Github linking](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue "Github linking ").
+
+## New Issue Creation Process:
+
+1. First, **_always verify_** the issue has **_not_** already been created in the repository.
+   - Failing to verify issues are only created once, almost always leads to duplicate issues and an unorganized repository.
+1. Create new issues using the appropriate [ISSUE_TEMPLATE](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository "Github Issue_Template docs").
```

Comment:
```suggestion
2. Create new issues using the appropriate [ISSUE_TEMPLATE](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository "Github Issue_Template docs").
```

Reply:
My bad, should've maybe asked. I don't remember where I picked it up, but if you number all your items in the numbered list with 1., no matter how many you add it will adjust to the correct number in the list. However it does not show up in the markdown. When looking at the md in preview mode, it does show the correct numbers. I'll go ahead and get them all changed back.